### PR TITLE
WIP: Add some items for mongod related specs

### PR DESCRIPTION
**Pull request by JoySnow**

### CODE REVIEW

File: insights/specs.py [hunk 1]
```diff
@@ -257,12 +257,14 @@ def docker_container_ids(broker):
 module = sf.listdir("/sys/module", name="module")
 modinfo = sf.foreach(module, "/usr/sbin/modinfo %s", name="modinfo")
 modprobe_conf = sf.simple_file("/etc/modprobe.conf", name="modprobe_conf", alias="modprobe.conf")
-sysconfig_mongod = sf.glob_file(["etc/sysconfig/mongod",
-                                 "etc/opt/rh/rh-mongodb26/sysconfig/mongod"],
+sysconfig_mongod = sf.glob_file(["/etc/sysconfig/mongod",
```
Comment (xiangce) [inline insights/specs.py]: Better reorder the specs with alphabetical order.

Comment (JoySnow) [reply insights/specs.py]: Yes, sure.

**Comment by JoySnow** [comment]
Related to mongod config.
This task is suspended.

**Comment by csams** [comment]
@JoySnow @xiangce Is this still relevant?

**Comment by JoySnow** [comment]
Since we have the changes at Branch: `ADD-few-mongod-specs_master`, it's okay for me to close this PR. I can redo these changes later when they are needed.

**Pull request closed without merging**